docs/react/use-with-create-react-app.en-US.md

```diff
@@ -146,8 +146,10 @@ module.exports = {
       plugin: CracoLessPlugin,
       options: {
         lessLoaderOptions: {
-          modifyVars: { '@primary-color': '#1DA57A' },
-          javascriptEnabled: true,
+          lessOptions: {
+            modifyVars: { '@primary-color': '#1DA57A' },
+            javascriptEnabled: true,
+          },
```

Comment:
Wrong usage, please check https://github.com/DocSpring/craco-less#configuration

<img width="636" alt="image" src="https://user-images.githubusercontent.com/507615/83596926-83055c00-a598-11ea-8091-34037ba8dd34.png">

Reply:
check https://github.com/DocSpring/craco-less/pull/42

Reply:
You are right.

Reply:
We should update https://ant.design/docs/react/use-in-typescript too.